storage/replica_test.go

```diff
@@ -665,7 +665,9 @@ func TestRangeTSCacheLowWaterOnLease(t *testing.T) {
 	tc.manualClock.Increment(int64(DefaultLeaderLeaseDuration + 1))
 	now := roachpb.Timestamp{WallTime: tc.manualClock.UnixNano()}
 
-	baseRTS, _ := tc.rng.tsCache.GetMax(roachpb.Key("a"), nil /* end */, nil /* txn */)
+	tc.rng.mu.Lock()
```

Comment:
yes, seems so.  tsCache was meant to be protected, but we didn't actually do so.